datamover/pkg/drivers/https/migration.go

```diff
@@ -610,7 +620,7 @@ func runjob(in *pb.RunJobRequest) error {
 					//update database
 					j.PassedCount = (int64(passedCount))
 					j.PassedCapacity = capacity
-					j.Progress = int64(capacity * 100 / j.TotalCapacity)
+					//j.Progress = int64(capacity * 100 / j.TotalCapacity)
 					logger.Printf("capacity:%d,TotalCapacity:%d Progress:%d\n", capacity, j.TotalCapacity, j.Progress)
```

Comment:
Why comment this line?

Reply:
Hi @wisererik . Thanks for the review. As we are calculating Progress code in progress function defined at the end, we will not need to calculate progress here. Thanks